lib/node_modules/@stdlib/math/base/special/bernoullif/src/main.c

```diff
@@ -0,0 +1,88 @@
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+#include "stdlib/math/base/assert/is_oddf.h"
+#include "stdlib/constants/float32/ninf.h"
+#include "stdlib/constants/float32/pinf.h"
+#include "stdlib/math/base/special/bernoullif.h"
+#include "stdlib/constants/float32/nan.h"
+
+static const float bernoulli_value[ 33 ] = {
+	1.00000000000000000000000000000000000000000f,
+  0.166666666666666666666666666666666666666667f,
+  -0.0333333333333333333333333333333333333333333f,
+  0.0238095238095238095238095238095238095238095f,
+  -0.0333333333333333333333333333333333333333333f,
+  0.0757575757575757575757575757575757575757576f,
+  -0.253113553113553113553113553113553113553114f,
+  1.16666666666666666666666666666666666666667f,
+  -7.09215686274509803921568627450980392156863f,
+  54.9711779448621553884711779448621553884712f,
+  -529.124242424242424242424242424242424242424f,
+  6192.12318840579710144927536231884057971014f,
+  -86580.2531135531135531135531135531135531136f,
+  1.42551716666666666666666666666666666666667e6f,
+  -2.72982310678160919540229885057471264367816e7f,
+  6.01580873900642368384303868174835916771401e8f,
+  -1.51163157670921568627450980392156862745098e10f,
+  4.29614643061166666666666666666666666666667e11f,
+  -1.37116552050883327721590879485616327721591e13f,
+  4.88332318973593166666666666666666666666667e14f,
+  -1.92965793419400681486326681448632668144863e16f,
+  8.41693047573682615000553709856035437430786e17f,
+  -4.03380718540594554130768115942028985507246e19f,
+  2.11507486380819916056014539007092198581560e21f,
+  -1.20866265222965259346027311937082525317819e23f,
+  7.50086674607696436685572007575757575757576e24f,
+  -5.03877810148106891413789303052201257861635e26f,
+  3.65287764848181233351104308429711779448622e28f,
+  -2.84987693024508822262691464329106781609195e30f,
+  2.38654274996836276446459819192192149717514e32f,
+  -2.13999492572253336658107447651910973926742e34f,
+  2.05009757234780975699217330956723102516667e36f,
+  -2.09380059113463784090951852900279701847092e38f
+};
+
+
+int32_t MAX_BERNOULLI = 64;
+
+/**
+* Computes the nth Bernoulli as a single-precision floating-point number.
+*
+* @param n    input value
+* @return     output value
+*
+* @example
+* float out = stdlib_base_bernoullif( 0 );
+* // returns 1
+*/
+float stdlib_base_bernoullif( const int32_t n ) {
+	if ( n < 0 ) {
+		return STDLIB_CONSTANT_FLOAT32_NAN; // NaN
+	}
+	if ( n == 1 ) {
+		return 0.5f;
+	}
+	if ( stdlib_base_is_oddf( n ) ) {
+		return 0.0f;
+	}
+	if ( n > MAX_BERNOULLI ) {
+		return ( ( n / 2 ) & 1 ) ? STDLIB_CONSTANT_FLOAT32_PINF : STDLIB_CONSTANT_FLOAT32_NINF;
+	}
+	return (float)bernoulli_value[ n / 2 ];
```

Comment:
```suggestion
	return bernoulli_value[ n / 2 ];
```
due to the earlier change the compiler was unable to read the 32 bernoulli values hence causing a CI test failure.
Pls do make this change and try again